codespell_lib/tests/test_dictionary.py

```diff
@@ -41,9 +39,9 @@
 
 def test_dictionaries_exist():
     """Test consistency of dictionaries."""
-    doc_fnames = set(op.basename(f[0]) for f in _fnames_in_aspell)
```

Comment:
This change kinda makes it less readable - have to look into the expression to see if it's a dict or set.

Reply:
I would revert these two lines in this merge request. Let's discuss them in #2085.
Thinking about it, `list()`/`set()` are easy to understand because they are English words, while the `[]`/`{}` "_syntax needs to be understood_". While both are acceptable, `[]`/`{}` reminds me of the arcane syntax of Perl. It reminds me that years ago, I would struggle to understand my own commented Perl scripts a few months after having written them, because of the syntax.